documentation/compose/kafka-ui-kraft-rest-proxy.yaml

```diff
@@ -0,0 +1,43 @@
+---
+version: '3.4'
+services:
+  # Based on https://github.com/confluentinc/kafka-images/blob/master/examples/confluent-local/docker-compose.yml but
+  # removed the env var defaults that work as-is with Kafka UI.
+  kafka0:
+    # Pre-configured image for Kraft mode and REST Proxy
+    # See: https://docs.confluent.io/platform/current/installation/docker/image-reference.html#ak-images
+    image: confluentinc/confluent-local:latest
+    hostname: kafka0
+    container_name: kafka0
+    ports:
+      - "9092:9092"
+      - "8082:8082"
+    environment:
+      # Normally you do not have to specify all these environment variables, but we are doing it to make kafka-ui work
+      # as the defaults are all localhost instead of a docker compose service name
+      # See: Defaults https://github.com/confluentinc/kafka-images/blob/master/local/include/etc/confluent/docker/configureDefaults
```

Comment:
I couldn't find a way to "hack" docker compose to treat kafka0 interactions from the other services as `localhost` (the default env vars in the image). So I ended up just overriding all ENV vars that had localhost in it that were required for Kafka UI and init kafka topics to work.